discovery-infra/add_triage_signature.py

```diff
@@ -0,0 +1,211 @@
+#!/bin/env python
+
+import argparse
+import logging
+import netrc
+import os
+import json
+from urllib.parse import urlparse
+import re
+from collections import OrderedDict
+import jira
+import requests
+from tabulate import tabulate
+
+
+logger = logging.getLogger(__name__)
+
+
+############################
+# Common functionality
+############################
+class Signature:
+    def __init__(self, jira_client, comment_identifying_string):
+        self._jclient = jira_client
+        self._identifing_string = comment_identifying_string
+
+    @staticmethod
+    def _get_metadata_json(cluster_url):
+        res = requests.get("{}/metdata.json".format(cluster_url))
+        res.raise_for_status()
+        return res.json()
+
+    def _find_signature_comment(self, key):
+        comments = self._jclient.comments(key)
+        for comment in comments:
+            if self._identifing_string in comment.body:
+                return comment
+        return None
+
+    def _update_triaging_ticket(self, key, comment, should_update=False):
+        report = "\n"
+        report += self._identifing_string
+        report += comment
+        comment = self._find_signature_comment(key)
```

Comment:
Maybe change to another variable name different than the parameter, it's confusing